phoenix-core-client/src/main/java/org/apache/phoenix/schema/TableProperty.java

```diff
@@ -336,6 +336,29 @@ public Object getValue(Object value) {
         @Override public Object getPTableValue(PTable table) {
             return table.getStreamingTopicName();
         }
+    },
+
+    MAX_LOOKBACK_AGE(PhoenixDatabaseMetaData.MAX_LOOKBACK_AGE, true, false, false) {
+        @Override
+        public Object getValue(Object value) {
+            if (value == null) {
+                return null;
+            }
+            else if (value instanceof Integer) {
+                return Long.valueOf((Integer) value);
+            }
+            else if (value instanceof Long) {
+                return value;
```

Comment:
What about -ve numbers?

Reply:
Negative numbers are not parsed by grammar as I tried and got error. As per grammar it seems like only whole numbers are expected.